library/x509_crt.c

```diff
@@ -1169,7 +1169,7 @@ int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path )
     if( dir == NULL )
         return( MBEDTLS_ERR_X509_FILE_IO_ERROR );
 
-#if defined(MBEDTLS_THREADING_PTHREAD)
+#if defined(MBEDTLS_THREADING_C)
```

Comment:
Could you please add a comment in the matching `#endif` as in https://github.com/ARMmbed/mbedtls/pull/739/files#diff-7eb92ee9f705f88338240a484631c19cR1216?